shared/fs/index.stories.js

```diff
@@ -81,7 +81,7 @@ const provider = Sb.createPropProviderWithCommon({
   }),
   ViewContainer: () => ({
     url: '/keybase/private/foo/bar.jpg',
-    mimeType: 'jpg',
+    mimeType: Constants.makeMime({mimeType: 'jpg'}),
```

Comment:
Did you set this to `jpg` instead of `image/jpeg` because otherwise it'd choose the image viewer?

Reply:
I set it with minimal changes to not be broken any more. Should it be `image/jpeg`?